src/proj.h

```diff
@@ -381,7 +380,9 @@ size_t proj_trans_generic (
     double *t, size_t st, size_t nt
 );
 
-int proj_trans_array (PJ *P, PJ_DIRECTION direction, size_t n, PJ_COORD *coord);
+
+/* Geographical to geocentric latitude */
+PJ_COORD proj_geoc_lat (PJ *P, PJ_DIRECTION direction, PJ_COORD coo);
```

Comment:
Is this really meant to go in the public API?

Reply:
Yes, that was my intention: We want to make it possible for an app to massage its input data into the form PROJ.4 expects, and we do not want the app to need to know which specific ellipsoid was selected (which can possibly have happened externally or through user input at a different access level).

Will update proj.def